### DISCUSSION TITLE
Peephole Camera - live stream

### DISCUSSION THREAD

**Discussion by N3orun** [tsightler/ring-mqtt#818] [Support Q&A]
Hi, im now trying for hours to grab a live stream from my Ring Peephole Video Doorbell Camera. 

Also tried via the Ring and the custom Ring integration, but all I see is a delayed and spead up version of a event stream - if at all - when using these. 

Entity-wise, I have a switch that toggles live video, but that results in this [behavior](https://share.digitalwhale.eu/yAuBd4b5)
I can click the Create Snapshot and display them, but I'm just not getting a live stream. 
Here are the relevant [entities](https://share.digitalwhale.eu/RBuk80L1)

Tried and failed with the [Video-Streaming-Guidelines](https://github.com/tsightler/ring-mqtt/wiki/Video-Streaming#home-assistant-generic-camera-configuration) , as well as a webrtc tutorial.

The ios App displays the livestream flawlessly. Running home assistant on a raspi, nothing out of the ordinary. 

Im thankful for any help or suggestions. 

Thanks!

**Comment by tsightler** [comment]
Did you look at the logs, what do they say?
